docs/src/main/sphinx/connector/prometheus.rst

```diff
@@ -91,6 +91,33 @@ Prometheus can be setup to require a Authorization header with every query. The
 ``prometheus.bearer.token.file`` allows for a bearer token to be read from the configured file. This file
 is optional and not required unless your Prometheus setup requires it.
 
+.. _Prometheus-type-mapping:
+
+Type mapping
+------------
+
+Because Trino and Prometheus each support types that the other does not, this
+connector modifies some types when reading or writing data. Data types may not
+map the same way in both directions between Trino and the data source. Refer to
+the following sections for type mapping in each direction.
+
+Prometheus type to Trino type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector maps Prometheus types to the corresponding Trino types following
+this table:
+
+.. list-table:: Prometheus type to Trino type mapping
+  :widths: 30, 20
+  :header-rows: 1
+
+  * - Prometheus database type
+    - Trino type
+  * - ``TIMESTAMP``
+    - ``TIMESTAMP``
+
+No other types are supported.
```

Comment:
Prometheus connector returns fixed columns. I think it would be better to document this information instead of type mapping.
```
  Column   |            Type             | Extra | Comment
-----------+-----------------------------+-------+---------
 labels    | map(varchar, varchar)       |       |
 timestamp | timestamp(3) with time zone |       |
 value     | double                      |       |
```

Sample rows:
```
trino:default> SELECT * FROM up;
                         labels                         |           timestamp            | value
--------------------------------------------------------+--------------------------------+-------
 {instance=localhost:9090, job=prometheus, __name__=up} | 2022-09-01 06:18:54.481 +09:00 |   1.0
 {instance=localhost:9090, job=prometheus, __name__=up} | 2022-09-01 06:19:09.446 +09:00 |   1.0
```

Reply:
I think we can compromise for Prometheus specifically, since there's still mapping between the Prometheus column types and Trino types we can keep the boilerplate header of "Type mapping" but adjust the content. We can instead introduce the table with something like:

> Prometheus uses static columns that have a defined mapping to Trino types according to the following table

Then the table uses our standard type mapping table format, but with headers like "Prometheus column" and "Trino type".

Reply:
This should also be directly under the "Type mapping" header, no need for an additional "Prometheus type to Trino type mapping" sub-header.